aptos-node/indexer-node.yaml

```diff
@@ -0,0 +1,42 @@
+base:
+    # Update this value to the location you want the node to store its database
+    data_dir: "/nodedata/data"
+    role: "full_node"
+    waypoint:
+        # Update this value to that which the blockchain publicly provides. Please regard the directions
+        # below on how to safely manage your genesis_file_location with respect to the waypoint.
+        from_file: "/node/waypoint.txt"
+
+execution:
+    # Update this to the location to where the genesis.blob is stored, prefer fullpaths
+    # Note, this must be paired with a waypoint. If you update your waypoint without a
+    # corresponding genesis, the file location should be an empty path.
+    genesis_file_location: "/node/genesis.blob"
+
+full_node_networks:
+    - discovery_method: "onchain"
+      # The network must have a listen address to specify protocols. This runs it locally to
+      # prevent remote, incoming connections.
+      listen_address: "/ip4/127.0.0.1/tcp/6180"
+      network_id: "public"
+      # Define the upstream peers to connect to
+      seeds:
+        {}
+
+storage:
+    enable_indexer: true
+    storage_pruner_config:
+        ledger_pruner_config:
+            enable: false
+
+indexer:
+    enabled: true
+    postgres_uri: "postgres://user:password@localhost:5432/indexer-db"
```

Comment:
Why hardcode this URL? Why not just keep it as `DATABASE_URL` and use the `sed` command to replace it?